partiql-lang/src/test/kotlin/org/partiql/lang/planner/transforms/AstToLogicalVisitorTransformTests.kt

```diff
@@ -822,7 +823,38 @@ class AstToLogicalVisitorTransformTests {
                                         ),
                                         lit(ionInt(2))
                                     )
+                                ),
+                                rowAlias = varDecl(AstToLogicalVisitorTransform.EXCLUDED)
+                            ),
+                            bag(
+                                struct(
+                                    structField(lit(ionString("id")), lit(ionInt(1))),
+                                    structField(lit(ionString("name")), lit(ionString("bob")))
                                 )
+                            )
+                        )
+                    }
+                }
+            ),
+            // Testing using excluded non-reserved keyword in condition
+            TestCase(
```

Comment:
Can we also add a test with a collection of values, e.g.:

```SQL
INSERT INTO foo AS f <<{'id': 1, 'name':'bob'}, {'id': 2, 'name':'Alice'}>> ON CONFLICT DO REPLACE EXCLUDED WHERE f.id > 2
```

I can tell that it passes but would be good to have it as part of the tests.
```